ci-operator/config/openshift/ingress-node-firewall/openshift-ingress-node-firewall-master.yaml

```diff
@@ -3,6 +3,10 @@ base_images:
     name: "4.12"
     namespace: ocp
     tag: base
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
```

Comment:
is this part standary for dev-scripts who creates this ns ?

Reply:
yes